src/uu/uniq/src/uniq.rs

```diff
@@ -340,6 +363,19 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
     0
 }
 
+fn get_delimiter(matches: &ArgMatches) -> Delimiters {
```

Comment:
please run rustfmt

Reply:
Oops :face_with_head_bandage: will fix.  (also configuring local pre-commit hook to avoid this in future)